src/test/regress/sql/multi_explain.sql

```diff
@@ -108,6 +108,10 @@ SELECT true AS valid FROM explain_xml($$
 SELECT true AS valid FROM explain_json($$
 	SELECT l_quantity FROM lineitem WHERE l_orderkey = 5$$);
 
+-- ensure EXPLAIN EXECUTE doesn't crash
+PREPARE router_executor_query AS SELECT l_quantity FROM lineitem WHERE l_orderkey = 5;
+EXPLAIN EXECUTE router_executor_query;
```

Comment:
It does but it feels a little weird that the tests are out of order just to minimize the size of the diff, really wish we had better tooling here. Moved it to the bottom and added some more kinds of queries.